mava/networks.py

```diff
@@ -93,22 +96,60 @@ def __call__(self, obs_embedding: chex.Array, observation: Observation) -> distr
                 `step_count`.
 
         Returns:
-            A distrax.Categorical distribution over the action space for sampling actions from.
+            A transformed tfd.categorical distribution on the action space for action sampling.
 
         NOTE: We pass both the observation embedding and the observation object to the action head
         since the observation object contains the action mask and other potentially useful
         information.
         """
 
-        actor_logits = nn.Dense(self.action_dim, kernel_init=orthogonal(0.01))(obs_embedding)
+        actor_logits = nn.Dense(self.env.action_dim, kernel_init=orthogonal(0.01))(obs_embedding)
 
         masked_logits = jnp.where(
             observation.action_mask,
             actor_logits,
             jnp.finfo(jnp.float32).min,
         )
 
-        return distrax.Categorical(logits=masked_logits)
+        #  We transform this distribution with the `Identity()` transformation to
+        # keep the API identical to the ContinuousActionHead.
+        return IdentityTransformation(distribution=tfd.Categorical(logits=masked_logits))
```

Comment:
The new continuous distribution has an entropy function 

```suggestion
        tfd.Categorical(logits=masked_logits)
```

Reply:
Yeah but doesn't require a seed attribute compared to what we do in Tanh distribution( used in dist.sample) 😞